src/main/resources/application.properties

```diff
@@ -22,6 +22,8 @@ spring.kafka.properties.security.protocol=SSL
 spring.kafka.ssl.key-store-location=${KEYSTORE_LOCATION:file://person-restapi-keystore/person-restapi.keystore.jks}
 spring.kafka.ssl.key-store-password=${PERSON_KEYSTORE_PASSWORD}
 spring.kafka.ssl.key-password=${PERSON_KEYSTORE_PASSWORD}
+spring.kafka.ssl.trust-store-location=${TRUSTSTORE_LOCATION:}
```

Comment:
By adding the trust store to the main properties, breaks the main envs (dev, test , prod). The truststore is only needed for localdev. If truststore pasword is set even an empty value results in spring wanting the location of a truststore.

Reply:
Well spotted @JWeston-HO ! I've addressed this, still without the need for a localdev profile. Please refer to the other PR (https://github.com/UKHomeOffice/callisto-localdev/pull/18) to see how this was implemented.
I've tested the branch pod on callisto-dev. Feel free to retest.